apps/pixeloven/site/src/server/views/Body/Body.tsx

```diff
@@ -0,0 +1,41 @@
+import { Script } from "@server/views";
+import { State } from "@shared/store/types";
+import React from "react";
+import { HelmetData } from "react-helmet";
+
+interface BodyProps {
+    children: string;
+    files?: {
+        css?: string[];
+        js?: string[];
+    };
+    initialState?: State;
+    helmet?: HelmetData;
+}
+
+function Body(props: BodyProps) {
+    // const bodyAttrs = props.helmet
```

Comment:
Do we need this again?

Reply:
I left it there because I am not sure why it suddenly started erroring out and I am also not sure whether we should be doing this or not.